packages/docs/command-line-interface/account.md

```diff
@@ -479,76 +479,6 @@ EXAMPLES
 
 _See code: [packages/cli/src/commands/account/new.ts](https://github.com/celo-org/celo-monorepo/tree/master/packages/cli/src/commands/account/new.ts)_
 
-### Offchain-read
```

Comment:
Why are all the following docs removed?

Reply:
turns out this was all called by `yarn build` failing earlier and resulting in contracts not being compiled. Will make a note of this in our own runbook/docs